Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>URL=\"jdbc:mysql://localhost:3306/\"\nDATABASE=\"shinobi_alliance_bank\"\nROOT_USER=\"root\"\nPASSWORD=\"\"\nUSER_TABLE=\"user\"\nBANK_TABLE=\"bank\"\nACCOUNT_TABLE=\"account\"\nTRANSACTION_TABLE=\"transactions\"\nUPI_TABLE=\"upi\"
===================================================================
diff --git a/.env b/.env
--- a/.env	
+++ b/.env	
@@ -6,4 +6,5 @@
 BANK_TABLE="bank"
 ACCOUNT_TABLE="account"
 TRANSACTION_TABLE="transactions"
-UPI_TABLE="upi"
\ No newline at end of file
+UPI_TABLE="upi"
+
